components/infobox/commons/infobox_widget_header.lua

```diff
@@ -104,7 +108,12 @@ function Header:_image(fileName, fileNameDark, default, defaultDark, size)
 	---@cast imageName -nil
 	local infoboxImageDark = Header:_makeSizedImage(imageName, fileNameDark or fileName, size, 'darkmode')
 
-	return mw.html.create('div'):node(infoboxImage):node(infoboxImageDark)
+	local imageTextNode = Header:_makeImageText(imageText)
+
+	return mw.html.create('div'):addClass('infobox-image-wrapper')
```

Comment:
`infobox-image-wrapper` class always getting set intended?
was it tested if that efects existing infoboxes?
(especiall those with fixed size, e.g. https://liquipedia.net/starcraft2/Stimpack)

Reply:
Should have no affect unless there's another position property (and yes it was tested)